Embedded CRT YouTube video on the page (#4727)
mattermost/docs: source/help/messaging/organizing-conversations.rst

Tutorial Video
---------------

> For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://youtu.be/hnD0Zj-mIbs>`__. 

Author: ```suggestion
For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/embed/hnD0Zj-mIbs>`__. 
```

> For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/embed/hnD0Zj-mIbs>`__. 

Author: ```suggestion
For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 
```

> For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/embed/hnD0Zj-mIbs>`__. 

Author: ```suggestion
For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://www.youtube.com/embed/hnD0Zj-mIbs>`__. 
```

> For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 

Author: ```suggestion
For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 
```

.. raw:: html

   <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">

>       <iframe src="https://youtu.be/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>

Author: ```suggestion
      <iframe src="www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```

>       <iframe src="www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>

Author: ```suggestion
      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```
